### PR TITLE
Opgops 1935

### PR DESCRIPTION
**Pull request by brettminnie**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+	@echo -e Not tagging repo
+endif
+
+pull:
+	for i in $(CORE_CONTAINERS) $(CHILD_CONTAINERS) $(LTS_CONTAINERS); do \
+			docker pull $(registryUrl)/$$i ; \
+	done
+
+showinfo:
+	@echo Registry: $(registryUrl)
```
Comment (tim-mcarthur-moj) [inline Makefile]: these vars are not all relevant any more

File: README.md [hunk 2]
```diff
@@ -1,2 +1,44 @@
...
+
+Why does every service creates /var/log structure at runtime?
+
+I't quite common that developer will run the container and mount /var/log
+So make sure that every service started will pre-create its log directory structure on start time.
+Instead of relying on RUN directives to do so.
+
+
+
+How to run application as specific user? Shall I use `su foo -c "ls"`?
```
Comment (tim-mcarthur-moj) [inline README.md]: please update to reflect our gosu usage.

File: opg-base-1604/docker/scripts/ebs-snapshot.sh [hunk 3]
```diff
@@ -0,0 +1,317 @@
+#!/bin/bash
```
Comment (tim-mcarthur-moj) [inline opg-base-1604/docker/scripts/ebs-snapshot.sh]: has this script been tested

File: opg-base-1604/docker/scripts/rds-snapshot.py [hunk 4]
```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
```
Comment (tim-mcarthur-moj) [inline opg-base-1604/docker/scripts/rds-snapshot.py]: has this script been tested

File: opg-base-1604/docker/confd/templates/beaver.conf.tmpl [hunk 5]
```diff
@@ -0,0 +1,11 @@
+[beaver]
+redis_url: redis://monitoring:6379/0
```
Comment (tim-mcarthur-moj) [inline opg-base-1604/docker/confd/templates/beaver.conf.tmpl]: set monitoring host to variable with default as `monitoring`

File: README.md (hunk 2 above)
Comment (brettminnie) [inline README.md]: Done

File: opg-base-1604/docker/scripts/ebs-snapshot.sh (hunk 3 above)
Comment (brettminnie) [inline opg-base-1604/docker/scripts/ebs-snapshot.sh]:
```
(docker-task-wrapper.sh) Initiating task: ebs_snapshot
(docker-task-wrapper.sh) Calling: /scripts/base/ebs-snapshot.sh --runmode single --metricpath sandbox.api.ebs.snapshots
[2017-05-22+10:40:26]: Running in single mode
[2017-05-22+10:40:26]: Recording to metric path xxx.xxx.ebs.snapshots
[2017-05-22+10:40:26]: Logging metrics to xxx:2003
[2017-05-22+10:40:26]: Snapshots are retained for 7 days
[2017-05-22+10:40:27]: Volume ID is vol-0d1...
[2017-05-22+10:40:28]: New snapshot is snap-025...
[2017-05-22+10:40:29]: Checking snap-025...
[2017-05-22+10:40:30]: Not deleting snapshot snap-025.... Description: [     "i-08c...-/dev/sda1-backup-2017-05-22" ]  ; Taken: Mon May 22 10:40:28 UTC 2017
[2017-05-22+10:40:30]: Checking snap-0ce...
[2017-05-22+10:40:31]: Not deleting snapshot snap-0ce... Description: [     "i-08c...-/dev/sda1-backup-2017-05-22" ]  ; Taken: Mon May 22 10:37:46 UTC 2017
[2017-05-22+10:40:31]: Sending metric: xxx.xxx.ebs.snapshots.volumes_total 1 1495449631
[2017-05-22+10:40:31]: Sending metric: sandbox.api.ebs.snapshots.volumes_failed 0 1495449631
[2017-05-22+10:40:31]: Sending metric: xxx.xxx.ebs.snapshots.volumes_success 1 1495449631
[2017-05-22+10:40:31]: Sending metric: xxx.xxx.ebs.snapshots.volumes_cleaned 0 1495449631
[2017-05-22+10:40:31]: Sending metric: xxx.xxx.ebs.snapshots.volumes_checksum 0 1495449631
Completed
```

File: Makefile (hunk 1 above)
Comment (brettminnie) [inline Makefile]: :+1:

**Pull request merged**